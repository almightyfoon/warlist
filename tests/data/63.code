c45b1b_-0g0x1a1v1v2DhC2t2b203g203f203gbBgr___-0whBd-d-1e1e3b1u1v1v1v2nf0f02B2Bgs