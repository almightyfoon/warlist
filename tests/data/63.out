https://conflictchamber.com/#c45b1b_-0gkv1a1v1v2DhC2t2b203g203f203gbBgr___-0whBd-d-1e1e3b1u1v1v1v2nf0f02B2Bgs

Cryx Army - 77 / 75 points
[Theme] The Ghost Fleet

!!! This army must contain 75 points or fewer

[Deneghra 1] Warwitch Deneghra [+26]
 - Defiler [8]
 - Slayer [10]
 - Slayer [10]
Captain Rengrave [0(5)]
Hellslinger Phantom [0(7)]
Pistol Wraith [0(5)]
Blackbane's Ghost Raiders (max) [17]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (3) [6]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (2) [4]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (3) [6]
Wraith Engine [15]
Armory (Objective) [0]

Cryx Army - 75 / 75 points
[Theme] Infernal Machines

[Mortenebra 2] Mortenebra, Numen of Necrogenesis [+27]
 - Inflictor [13]
 - Inflictor [13]
 - Ripjaw [7]
 - Ripjaw [7]
 - Scavenger [7]
 - Seether [13]
 - Slayer [10]
 - Slayer [10]
 - Slayer [10]
Necrotech [2]
Soul Trapper [1]
Soul Trapper [1]
Warwitch Siren [4]
Warwitch Siren [4]
Bunker (Objective) [0]
